pomodoro/urls.py

```diff
@@ -18,10 +18,13 @@
 from django.contrib.auth import views as auth_views
 
 from pages.views import home_view
-from accounts.views import create_account_view
 
+from accounts.views import create_account_view
+from timer.views import index_view
+  
 urlpatterns = [
     path('', home_view, name='home'),
+    path('index/', index_view, name ='index')
```

Comment:
needs comma at end

Reply:
I added in the comma, thanks for catching that!